hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableAccountStoreImpl.java

```diff
@@ -132,6 +135,33 @@ protected Account getAccountLeaf(@NonNull final AccountID id) {
                         .build());
     }
 
+    /**
+     * Returns the {@link AccountID} referenced by the given alias, whether it is a direct reference or
+     * an "indirect" reference by a proto ECDSA key whose implied EVM address is the alias.
+     *
+     * @param alias the alias to look up
+     * @return the account id referenced by the alias, or null if not found
+     */
+    private @Nullable AccountID unaliasWithEvmAddressConversionIfNeeded(@NonNull final Bytes alias) {
+        final var maybeDirectReference = aliases.get(new ProtoBytes(alias));
+        if (maybeDirectReference != null) {
+            return maybeDirectReference;
+        } else {
+            try {
+                final var protoKey = Key.PROTOBUF.parseStrict(BufferedData.wrap(alias.toByteArray()));
```

Comment:
Nit: there is an API on bytes to get a readable stream, and you can pass that in, rather than converting to byte array and then wrapping it.